debs-to-download

```diff
@@ -1,3 +1,5 @@
 logi-rts
+revpipycontrol
```

Comment:
Replaced by revpicommander.
Should we remove this completely for new images?

Reply:
Does revpicontrol still work? Are there any users of revpicontrol? This might break there workflows?

Reply:
It still works. Typically, you install it on your development machine and connect over the network - This machines stay untouched. If the image is flashed and users really use it locally in the GUI, they can switch directly to RevPiCommander. It looks the same, has more features and will be further developed.

Reply:
@iluminat23 Ihmo this change wouldn't break much / anything as we're not removing the packages from our repo, but only remove them from our default installation. If a customer needs them for whatever use-case, they can install them from our repo. Nevertheless we would need to state this in our release notes.